Integrate-Kobiton.md

```diff
@@ -0,0 +1,163 @@
+
+To get your Kobiton Username and API Key, follow instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 2. Samples
+In this repository, we have already provided samples for executing automation test in Kobiton:
+- Script for executing automation test on Kobiton iOS devices (`/samples/ios-app-test.js`).
+- Script for executing automation test on Kobiton Android devices (`/samples/android-app-test.js`).
+- CircleCI configuration file (`/.circleci/config.yml`).
+
+Link to the sample(s): https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative/tree/master/samples
+
+In this guide, as we will be using the provided example, therefore, follow these steps below:
+
+1. Fork this repository https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative
+
+2. Synchronize the forked repository with CircleCI. If you don't know how to do it, refer to instruction at [CircleCI document](https://circleci.com/docs/2.0/getting-started/)
+
+3. Clone the forked repository as we will be modifying it later.
+
+## B. Setup
+### 1. Setting Kobiton Username and API key
+
+In your project configuration in CircleCI, add these two environment variables :
+- KOBITON_USERNAME - Your Kobiton's username.
+- KOBITON_API_KEY - Your Kobiton's API Key.
+
+Your `Environment Variables` window should look like this
+
+![Environment variables](./assets/env-variables-after-add.png)
+
+### 2. Getting Kobiton device desired capabilities
+In order to interact with a specific device in Kobiton, its corresponding desired capabilities needs to be supplied.
+
+To get desired capabilities of a Kobiton device, follow these steps below :
+
+1. Go to [https://portal.kobiton.com/login](https://portal.kobiton.com/login) and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run your test script on, click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the popup window, you can choose the language of your script, application testing type, etc. In this example, we will be using NodeJS, therefore, change `Language` section to `NodeJS`. Then, change the value of `App Type` to `Hybrid/Native from Apps`. The desired capabilities code will be generated and displayed in the right pane of the popup (marked **blue**).
+
+![Automation Settings](./assets/desired-caps.jpg)
+
+5. Write down the generated code as we will be using it later.
+
+### 3. Configuring automation test script
+
+In the `samples` folder, we have provided two sample scripts for executing automation test on Kobiton iOS and Android devices. In those provided scripts, we will be executing automation testing of `ApiDemos-debug` application on Android and `UIKitCatalog-Test-Adhoc` application on iOS.
```

Comment:
"In those provided scripts, we will be executing automation testing of `ApiDemos-debug` application on Android and `UIKitCatalog-Test-Adhoc` application on iOS." I think this is a distraction information, remove this